recipe/meta.yaml

```diff
@@ -1,47 +1,46 @@
-{% set version = "1.12.0" %}
+{% set version = "1.13.0" %}
 
 package:
   name: geoviews-core
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/g/geoviews/geoviews-{{ version }}.tar.gz
-  sha256: e2cbef0605e8fd1529bc643a31aeb61997f8f93c9b41a5aff8b2b355a76fa789
+  sha256: 7554a1e9114995acd243546fac6c6c7f157fc28529fde6ab236a72a6e77fe0bf
 
 build:
   number: 0
 
 outputs:
   - name: geoviews-core
-    script: build_base.sh
     build:
       noarch: python
+      script: python -m pip install . --no-deps -vv
```

Comment:
I see Panel doesn't have the same script command:

https://github.com/conda-forge/panel-feedstock/blob/main/recipe/meta.yaml#L17

Reply:
I'm pretty sure `{{ PYTHON }}` is unavailable when `build` is inside `outputs`. I can give it a go if you want.